teachertool/src/components/makecodeFrame.tsx

```diff
@@ -0,0 +1,170 @@
+/// <reference path="../../../built/pxteditor.d.ts" />
```

Comment:
For this file (and components in general), it might be good to keep the capitalize-the-first-letter naming convention (i.e. MakeCodeFrame.tsx). It's a little weird/inconsistent with our naming for other files, but personally I think it's nice for the file to map to the component name exactly.

Reply:
I will do this as a last step since when I do this, it will say there's a new file. I think that would be annoying to try to read through when looking at my other new updates.